app/views/registrations/school_not_eligible/show.html.erb

```diff
@@ -1,8 +1,12 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% content_for :before_content, govuk_back_link(
+      text: "Back",
+      href: registrations_school_profile_path)
+    %>
     <h2 class="govuk-heading-l">Sorry, your school is not eligible for the ECF</h2>
-    <p>Only schools in England are eligible for the ECF.</p>
-    <p>The following institutions are also not eligible:</p>
+    <p class="govuk-body">Only schools in England are eligible for the ECF.</p>
+    <p class="govuk-body">The following institutions are also not eligible:</p>
```

Comment:
Does that change the style?

I thought we add `govuk-body` to our main element
```
      <main class="govuk-main-wrapper govuk-body" id="main-content" role="main">
```
so that we don't have to add `govuk-body` everywhere.

Reply:
I noticed the designer did have govuk-body on every paragraph in his prototype. I haven't seen it make any difference so you're probably correct that it's not necessary in the paragraphs